flink-yarn/src/main/java/org/apache/flink/yarn/Utils.java

```diff
@@ -161,14 +161,9 @@ public static void setupYarnClassPath(Configuration conf, Map<String, String> ap
 
 		fs.copyFromLocalFile(false, true, localSrcPath, dst);
 
-		// Note: If we used registerLocalResource(FileSystem, Path) here, we would access the remote
```

Comment:
This comment basically explains why we cannot just query the timestamp from remote filesystem, like this:
`fs.getFileStatus(dst)`. You are also trying to basically revert the fix introduced in [FLINK-8801](https://issues.apache.org/jira/browse/FLINK-8801).

Reply:
ok, thanks @azagrebin .